icu4c/source/i18n/unicode/simplenumberformatter.h

```diff
@@ -68,44 +68,49 @@ class U_I18N_API SimpleNumber : public UMemory {
      */
     void multiplyByPowerOfTen(int32_t power, UErrorCode& status);
 
-#ifndef U_HIDE_DRAFT_API
     /**
-     * Rounds the value currently stored in the SimpleNumber to the given power of 10.
+     * Rounds the value currently stored in the SimpleNumber to the given power of 10,
+     * which can be before or after the decimal separator.
      *
-     * This function immediately mutates the inner value.
+     * This function does not change minimum integer digits.
      *
-     * @draft ICU 73
+     * @stable ICU 73
      */
     void roundTo(int32_t power, UNumberFormatRoundingMode roundingMode, UErrorCode& status);
 
+#ifndef U_HIDE_DRAFT_API
     /**
-     * Truncates the most significant digits to the given maximum number of integer digits.
+     * Sets the number of integer digits to the given amount, truncating if necessary.
      *
-     * This function immediately mutates the inner value.
+     * @draft ICU 75
+     */
+    void setMaximumIntegerDigits(uint32_t maximumIntegerDigits, UErrorCode& status);
+#endif // U_HIDE_DRAFT_API
+
+#ifndef U_HIDE_DEPRECATED_API
+    /** Alias for setMaximumIntegerDigits.
      *
-     * @draft ICU 73
+     * @deprecated ICU 75
      */
-    void truncateStart(uint32_t maximumIntegerDigits, UErrorCode& status);
+    void truncateStart(uint32_t maximumIntegerDigits, UErrorCode& status) {
+        setMaximumIntegerDigits(maximumIntegerDigits, status);
```

Comment:
This does not compile if U_HIDE_DRAFT_API but not U_HIDE_DEPRECATED_API.
Please try to move line 88 `#endif // U_HIDE_DRAFT_API` to after line 98 `#endif // U_HIDE_DEPRECATED_API`